src/stories/AmplienceImageStudio.stories.ts

```diff
@@ -4,10 +4,15 @@ import {
   AmplienceImageStudio,
   AmplienceImageStudioOptions,
 } from '../AmplienceImageStudio';
-import { ImageStudioReason, SDKImage } from '../types';
+import {
+  ImageSaveEventData,
+  ImageStudioEventType,
+  SDKEventType,
+  SDKImage,
+} from '../types';
 
-const IMAGE_STUDIO_DOMAIN = 'https://app.amplience.net';
-// const IMAGE_STUDIO_DOMAIN = 'http://localhost:5173';
+// const IMAGE_STUDIO_DOMAIN = 'https://app.amplience.net';
```

Comment:
The legacy tests have this at `https://app.amplience.net`, but these ones have been switched to localhost. I guess they should be consistent?